doc_source/security_iam_resource-based-policy-examples.md

```diff
@@ -85,7 +89,11 @@ Every user of AWS Elemental MediaConnect must have a policy that defines permiss
                 "iam:PassRole"
             ],
             "Effect": "Allow",
-            "Resource": "*"
+            "Resource": "*",
+            "Condition": {
+                "StringLike": {
+                    "iam:PassedToService": "mediaconnect.amazonaws.com"
```

Comment:
This block is missing the closed curly brackets

Reply:
Done. Validated the json this time. Sorry about that miss!